mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -218,7 +218,8 @@
         "MOM_MF_CosLvlGain" "(%s1 level(s) gained!)"
         "MOM_MF_XPGainRank" "Rank XP Gained: %s1"
         "MOM_MF_PlayReplay" "Play Replay"
-        "MOM_MF_DetachMouse" "Right click to enable the mouse!"
+        "MOM_MF_MouseInactive" "Right click to enable the mouse!"
+        "MOM_MF_MouseActive" "Right click to disable the mouse!"
```

Comment:
Now I'm starting to think this can just be one label:
```suggestion
        "MOM_MF_MouseToggle" "Right click to toggle mouse control!"
```

Reply:
Good idea. Also don't need to worry about dynamically setting the label text now. Removed a useless override to `SetMouseInputEnabled` function thanks to that.